dashboard/test/models/script_test.rb

```diff
@@ -327,7 +327,7 @@ def populate_cache_and_disconnect_db
   end
 
   test 'level_concept_difficulty uses preloading' do
-    level = Script.find_by_name('20-hour').script_levels.third.level
```

Comment:
Because that level has no level_concept_difficulties, so `assert_equal expected, level.level_concept_difficulty` also generated a "use `assert_nil` instead" warning.